azure/disk.sh

```diff
@@ -1,5 +1,60 @@
 # Disk related functions
 
+# azure_disk_new creates a new instance of "managed disk".
+# `name` is the name of the managed disk.
+# `group` is name of resource group.
+# `location` is the Azure Region.
+fn azure_disk_new(name, group, location) {
+	instance = (
+		"--name"
+		$name
+		"--resource-group"
+		$group
+		"--location"
+		$location
+	)
+
+	return $instance
+}
+
+# azure_disk_set_size sets the size of "managed disk".
+# `instance` is the name of the instance.
+# `size` is the size in Gb of managed disk.
+fn azure_disk_set_size(instance, size) {
+	instance <= append($instance, "--size-gb")
+	instance <= append($instance, $size)
+
+	return $instance
+}
+
+# azure_disk_set_sku sets the kind of "managed disk".
+# `instance` is the name of the instance.
+# `sku` is the underlying storage sku. Allowed values: Premium_LRS,
+#  Standard_LRS. Default: Premium_LRS.
```

Comment:
I don't think is good to document what's default. Azure could change default in the future and then we'll have a wrong documentation. Maybe say that if this function isn't used then it will default to azure default underlying storage.